tests/acceptance/features/webUIRenameFiles/renameFiles.feature

```diff
@@ -40,17 +30,6 @@ Feature: rename files
     Examples:
       | from_name                               | to_name                               |
       | "'single'quotes.txt"                    | "single-quotes.txt"                   |
-
-  # Merge with scenario above once the issue is resolved
-  Scenario Outline: Rename a file that has special characters in its name
-    Given the user has created file "sämple,1.txt"
-    And the user has reloaded the current page of the webUI
```

Comment:
These steps will be needed in the scenario above.
Or we could add `sämple,1.txt` to the `webUISkeleton` in the testing app.